### DISCUSSION TITLE
many to many relationship

### DISCUSSION THREAD

**Discussion by champkris** [thedevdojo/voyager#5219] [General]
I'm trying to have my Post be able to belong to Many Category (Many to Many relationship) via Voyager. Still not quite get it right. Any tips would be appreciated.

**Comment by MrCrayon** [comment]
First tip read Voyager documentation:
https://voyager-docs.devdojo.com/bread/relationships
Second tip, if you are not sure how relationships work read Laravel documentation:
https://laravel.com/docs/8.x/eloquent-relationships
Third tip when you ask a question be more specific, when you say: _"Still not quite get it right"_ we have no way to tell which problem you are facing.
